vnext/src/Libraries/Components/Flyout/Flyout.js

```diff
@@ -3,14 +3,13 @@
  * Licensed under the MIT License.
  * @format
  */
```

Comment:
Should this have an `@flow strict` or `@flow strict-local`?

Reply:
Probably?